cmd/limactl/start.go

```diff
@@ -376,6 +377,9 @@ func chooseNextCreatorState(st *creatorState, yq string) (*creatorState, error)
 			},
 		}
 		if err := survey.AskOne(prompt, &ans); err != nil {
+			if err == terminal.InterruptErr {
+				logrus.Fatal("interrupted")
```

Comment:
Maybe
```suggestion
				logrus.Fatal("terminal was interrupted")
```

Reply:
I think that the "more correct" message probably would be:

`Interrupted by user`

WDYT?

Reply:
SGTM